### PR TITLE
Change to no longer check stage of proposal tracking transaction

### PR DESCRIPTION
**Pull request by RainFallsSilent**

### CODE REVIEW

File: blockchain/blockchain.go [hunk 1]
```diff
@@ -506,9 +506,12 @@ func (b *BlockChain) CreateCRAssetsRectifyTransaction() (*Transaction, error) {
 	if err != nil {
 		return nil, err
 	}
-	if len(utxos) < int(b.chainParams.MaxCRAssetsAddressUTXOCount) {
+	if len(utxos) < int(b.chainParams.MinCRAssetsAddressUTXOCount) {
```
Comment (Houshoupei84) [inline blockchain/blockchain.go:509]: Here should be less than MinCRAssetsAddressUTXOCount.

File: blockchain/txvalidator.go [hunk 2]
```diff
@@ -2043,11 +2043,17 @@ func (b *BlockChain) checkCRCProposalRealWithdrawTransaction(txn *Transaction,
...
 	// Inputs count should be greater than or equal to MaxCRAssetsAddressUTXOCount
-	if len(txn.Inputs) < int(b.chainParams.MaxCRAssetsAddressUTXOCount) {
+	if len(txn.Inputs) > int(b.chainParams.MaxCRAssetsAddressUTXOCount) {
 		return errors.New("inputs count should be greater than or " +
 			"equal to MaxCRAssetsAddressUTXOCount")
 	}
 
+	// Inputs count should be less than or equal to MinCRAssetsAddressUTXOCount
+	if len(txn.Inputs) < int(b.chainParams.MinCRAssetsAddressUTXOCount) {
+		return errors.New("inputs count should be less than or " +
```
Comment (Houshoupei84) [inline blockchain/txvalidator.go:2053]: May need remove or equal to

File: cmd/common/flags.go [hunk 3]
```diff
@@ -371,6 +371,10 @@ var (
 		Name:  "maxcrassetsaddressutxocount",
 		Usage: "defines the maximum number of utxo cr assets address can have ",
 	}
+	MinCRAssetsAddressUTXOCount = cli.StringFlag{
+		Name:  "mincrassetsaddressutxocount",
+		Usage: "defines the maximum number of utxo cr assets address can have ",
```
Comment (Houshoupei84) [inline cmd/common/flags.go:376]: should be minimum

Comment (RainFallsSilent) [reply cmd/common/flags.go]: fixed

File: blockchain/blockchain.go (hunk 1 above)
Comment (RainFallsSilent) [reply blockchain/blockchain.go:509]: fixed

File: blockchain/txvalidator.go (hunk 2 above)
Comment (RainFallsSilent) [reply blockchain/txvalidator.go]: fixed

**Pull request merged**